pkg/lockfile/lastwrite.go

```diff
@@ -19,6 +20,12 @@ type LastWrite struct {
 	state []byte // Contents of the lock file.
 }
 
+var (
+	lockFiles           map[string]*LockFile
+	lockFilesLock       sync.Mutex
```

Comment:
Non-blocking: `lastWriterIDCounter` relates to `LastWrite`, … but these two do belong back with the only user in `lockfile.go` :)